CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Added
+* Use `https` as the default protocol when constructing the `tokenServicesUrl` in the `featureServerRestInfo` handler. Default is overridden when an `ssl` property is defined on the result of `authenticationSpecification()`
```

Comment:
this is going to require additional documentation and probably code samples for how to run koop w/ https support

Reply:
I agree.  I hesitated to add to the koop-output-geoservices README, as we haven't put much there in the past.

Reply:
Yeah I think this needs to go on the documentation site and have links in output-geoservices and sample-provider